classes/fieldset/field.php

```diff
@@ -583,7 +583,14 @@ protected function template($build_field)
 
 		if (is_array($build_field))
 		{
-			$label = $this->label ? str_replace('{label}', $this->label, $form->get_config('group_label', '<span>{label}</span>')) : '';
+            $tagLabel = '{label}';
+            if(is_array($this->label)) {
+                $tagLabel = array('{label}');
+            }
+
+            $labelName = $this->label ? str_replace($tagLabel, $this->label, $form->get_config('group_label', '<span>{label}</span>')) : '';
+            $label = $form->label($labelName, null, array('id' => 'label_'.$this->name));
```

Comment:
exemple of field config : 

``` 'prof_type' => array( 
         'label' => __('Je suis').'<span class="required">*</span>',
            'form' => array(
                'type' => 'radio',
                'value' => 'journaliste',
                'options' => array(
                    'journaliste' => __('Journaliste'),
                    'pro' => __('Professionnel'),
                ),
            ),
            'validation' => array('required'),
        ),```

when we are `validation`, `$this->label` is an array.